src/routes/v1/registry/authorization/handler.js

```diff
@@ -8,10 +8,10 @@ import {
 import log from "logger";
 import { createDockerJWT } from "registry/jwt";
 import validateDeploymentCredentials from "deployments/validate/authorization";
-import { ACTIONS, VALID_RELEASE_NAME } from "deployments/validate/docker-tag";
+import { ACTIONS } from "deployments/validate/docker-tag";
 import config from "config";
 import { compact, isArray, find, includes } from "lodash";
-import { ENTITY_DEPLOYMENT } from "constants";
+import { ENTITY_DEPLOYMENT, VALID_RELEASE_NAME } from "constants";
```

Comment:
```suggestion
import { ENTITY_DEPLOYMENT, VALID_DOCKER_IMAGE_NAME } from "constants";
```

Plus the usage in later in the file, which I can't do as a suggestion.